addons/sys_radio/stringtable.xml

```diff
@@ -188,12 +200,18 @@
         </Key>
         <Key ID="STR_ACRE_sys_radio_ui_CurrentChannelBlock">
             <English>Current Channel Block</English>
+            <German>Aktueller Kanalblock</German>
+            <Italian>Blocco del canale corrente</Italian>
```

Comment:
A `channel block` comprises multiple channels, so `channel` should have a plural meaning, no?
If so, I think this fits better.
```suggestion
            <Italian>Blocco di canali corrente</Italian>
```

Reply:
That was stupid...